CHANGES.md

```diff
@@ -1,3 +1,18 @@
+Synapse 1.24.0 (2020-12-00)
```

Comment:
Apart from the version change, I might also recommend changing `2020-12-00` to `2020-12-XX` just to make it slightly more obvious when it comes time to release :slightly_smiling_face: